lib/services/ios-debug-service.ts

```diff
@@ -103,11 +103,18 @@ class IOSDebugService extends DebugServiceBase implements IPlatformDebugService
 
 		let lineStream = byline(child_process.stdout);
 		this._childProcess = child_process;
+		const pidRegExp = new RegExp(`${debugData.applicationIdentifier}:\\s?(\\d+)`);
```

Comment:
Nevermind - I've created a new helper method and added unit tests for it: https://github.com/telerik/mobile-cli-lib/pull/942